tests/test_global_settings.txt

```diff
@@ -0,0 +1,522 @@
+bridge_skin_material_flow_3,110

```

Comment:
Where do these settings come from?

Are these e.g. UM2 PLA settings? (That's a common default settings set)

Why not load in setting values from a json file?

Minor suggestions:
- use `=` instead of `,` just like command line parsing and for readibility
- escape newline characters in gcode settings (e.g. start_gcode)

Reply:
It looks like it's an UM3, taken by the Griffin g-code flavour and the machine width. I guess it was not a JSON file since CSV is simpler to parse even if it's a custom implementation.

I've implemented your minor suggestions in 1898240d4384d56218617c15521342b218e90842 and 781eb75147c2565fb3f2a347047cf643003ba714.